scheduler/jobs/scheduleMessage.js

```diff
@@ -0,0 +1,28 @@
+const source = require('rfr');
+
+const logger = source('bot/utils/logger');
+const { sendMessage } = source('bot/utils/util');
+const Bot = source('bot');
+
+function handleSendMessage(job) {
```

Comment:
Imo rename the function sendMessage.js

Reply:
wontfix because sendMessage already exists, but I did rename it in 3b66bcf41c99fa3390cf7d0c2bca01618b295549